TiFBackendUtils/index.ts

```diff
@@ -1,12 +1,17 @@
+import "TiFShared/lib/Math.js"; // needed to compile.
```

Comment:
Typescript compiler did not recognize extensions from TiFShared, so I needed to manually add these imports